src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label class="Formfield">Surname:
+          <input
+          type="text"
+           name="Surname"
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="Formfield">
+          Name:
+          <input
+          type="text"
+           name="Name"
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="Formfield">
+          How old are You?
+          <input type="number" name="Old"
+          min="1"
+          max="100"
+          value="12"
+          >
```

Comment:
```suggestion
          <input
            type="number"
            name="Old"
            min="1"
            max="100"
            value="12"
          >
```

![image](https://user-images.githubusercontent.com/80700247/222894696-b4dcbc45-ac89-4df9-ae3e-23c11dbd6abc.png)